src/CommunityToolkit.Maui.Core/Capabilities/StatusBar/StatusBar.android.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Diagnostics;
+using Android.OS;
+using Android.Util;
+using Android.Views;
+using Microsoft.Maui.Platform;
+using Activity = Android.App.Activity;
+using Debug = System.Diagnostics.Debug;
+
+namespace CommunityToolkit.Maui.Core.Capabilities;
+static partial class StatusBar
+{
+	static void PlatformSetColor(Color color)
+	{
+		if (!IsSupported())
+		{
+			return;
+		}
+		Activity.Window?.SetStatusBarColor(color.ToPlatform());
+	}
+
+	static void PlatformSetStyle(StatusBarStyle style)
+	{
+		if (!IsSupported())
+		{
+			return;
+		}
+
+		switch (style)
+		{
+			case StatusBarStyle.DarkContent:
+				AddBarAppearanceFlag(Activity, (StatusBarVisibility)SystemUiFlags.LightStatusBar);
+
+				break;
+			default:
+				RemoveBarAppearanceFlag(Activity, (StatusBarVisibility)SystemUiFlags.LightStatusBar);
+				break;
+		}
+	}
+
+
+	static void AddBarAppearanceFlag(Activity activity, StatusBarVisibility flag) =>
+			SetBarAppearance(activity, barAppearance => barAppearance |= flag);
```

Comment:
nit: Remove one space
```suggestion
		SetBarAppearance(activity, barAppearance => barAppearance |= flag);
```